### PR TITLE
Review and update changes in "tikv benchmark overview #271"

### PR DESCRIPTION
**Pull request by qiancai**
Signed-off-by: qiancai <qqzczy@126.com>

### What is changed?

In this PR, I've reviewed and updated doc changes made in https://github.com/tikv/website/pull/271 from the user perspective.

### CODE REVIEW

File: content/docs/5.1/deploy/performance/overview.md [hunk 1]
```diff
@@ -1,58 +1,62 @@
 ---
-title: Overview
-description: How to make a benchmark over a TiKV cluster
+title: Performance Overview
+description: An overview of TiKV benchmark performance
```
Comment (iosmanthus) [inline content/docs/5.1/deploy/performance/overview.md:3]:
```suggestion
description: An overview of TiKV performance
```

**Comment by iosmanthus** [comment]
@qiancai Rest LGTM!

File: content/docs/5.1/deploy/performance/overview.md (hunk 1 above)
Comment (qiancai) [reply content/docs/5.1/deploy/performance/overview.md]: Done.

**Review by iosmanthus — APPROVED** [review]
LGTM

**Pull request merged**